packages/playground/src/views/solutions_view.vue

```diff
@@ -5,8 +5,31 @@
       <v-card-title class="pa-0">Labs</v-card-title>
     </v-card>
     <v-text-field label="Search for a lab" v-model="searchItem" class="mb-5" clearable></v-text-field>
-    <ApplicationCards :cards="filteredCards" />
-    <p v-if="filteredCards.length === 0" class="mx-3 mb-3">No solution was found with the provided search query.</p>
+    <p v-if="filteredAppCards.length === 0 && filteredOrchCards.length === 0" class="mx-3 mb-3">
+      No solution was found with the provided search query.
+    </p>
+
+    <v-card class="my-5 pa-5" v-if="filteredOrchCards.length > 0"
+      ><v-card-title
+        class="mb-2"
+        :style="{
+          fontSize: '1.25rem',
+        }"
+        >Orchestrators</v-card-title
+      >
+      <ApplicationCards :cards="filteredOrchCards" />
+    </v-card>
+    <v-divider class="my-4"></v-divider>
```

Comment:
The divider is not necessary. Both cards are separated already, and it's shown in case no solutions are found. Can we remove it?

![image](https://github.com/user-attachments/assets/1c67a566-e062-4173-a5af-aee4421a2ac7)

Reply:
good catch, fixed, please check